### PR TITLE
pin sanic-plugins-framework to 0.8.2

### PR DESCRIPTION
**Pull request by wochinge**
**Proposed changes**:
- backport of https://github.com/RasaHQ/rasa/pull/5064

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -41,6 +41,7 @@ terminaltables==3.1.0
 sanic==19.9.0
 sanic-cors==0.9.9.post1
 sanic-jwt==1.3.2
+sanic-plugins-framework==0.8.2
```
Comment (tabergma) [inline requirements.txt]: Do you also want to add a link to the PR?

Comment (wochinge) [reply requirements.txt]: to which pr? added one in the description of this pr

Comment (tabergma) [reply requirements.txt]: @federicotdn asked me to add `# https://github.com/RasaHQ/rasa/pull/5064` before that line.

Comment (wochinge) [reply requirements.txt]: done 👍

**Pull request merged**